test/Serilog.Tests/Parsing/MessageTemplateParserTests.cs

```diff
@@ -20,12 +21,38 @@ static void AssertParsedAs(string message, params MessageTemplateToken[] message
                 messageTemplateTokens);
         }
 
+        [Fact]
+        public void ANullMessageIsAException()
```

Comment:
`MessageTemplateIsRequired()` might be more consistent with naming of other similar tests.